### ISSUE TITLE
Feature Request: Replace R with Python

### ISSUE THREAD

**Issue by marcpare** [SUMSarizer/SUMSarizer#32]
How close could a Python-based ML stack get to R's accuracy and performance? This would make it much easier to set up the app. It would also make the ML worker cleaner.

**Comment by bartholmberg** [comment]
Is there an equivalent Python based super-learner algorithm?   Has D. Wilson commented on this?

**Comment by marcrecurve** [comment]
Yes, but we haven't tried it out yet. And, yes, we've talked about it before.
